ui/src/shared/middleware/resizeLayout.js

```diff
@@ -1,4 +1,5 @@
 // Trigger resize event to relayout the React Layout plugin
+import {enablePresentationMode} from '../actions/app'
```

Comment:
Nit-picking, but personally I prefer that we do away with relative paths and instead always import via the top-level aliased `shared` (or `src`, etc. where applicable), even though it's in the same directory.

Reply:
oh yeah, for sure, that was actually do to an Atom extension I used that automatically finds modules, but it's not configured to use webpack paths. I'll fix that.